src/pages/collection/series/SeriesImages.tsx

```diff
@@ -96,7 +97,16 @@ const SeriesImages = () => {
           <Button
             buttonType="primary"
             className="rounded-md border border-panel-border px-4 py-3 font-semibold"
-            disabled
+            disabled={!Object.keys(selectedImage).length}
```

Comment:
Also, we should also probably disable the button if it's already the default/preferred image. That, or turn it into a 'Remove As Default Series Poster'

Reply:
I agree

Reply:
Disabling the button makes more sense than changing the text/option to remove.

Reply:
> > Disabling the button makes more sense than changing the text/option to remove.
> 
> How do we reset it back to default then?

Honestly, I don't see a reason why we need an option to change back to the default. If you're unhappy with the current image, you can just change it to whichever makes you happy.

Reply:
Unmarking makes sense for the cases where a random poster is returned. Those will always return the default if one is set, instead of random

Reply:
IMO, this should work fine. 

https://github.com/ShokoAnime/Shoko-WebUI/assets/9443295/b49bceec-2602-4bcb-b962-e222921017b7

Reply:
How does that fix what da3d mentioned?

Reply:
Lets clarify something here.
 
Currently, our random poster implementation does not update the default poster. So, a group, which normally shows the main series poster will have a random poster selected and when you go inside that group and depending on the number of posters you have, it will also be different. However, inside the series, it always shows the default poster. 

I'm not here to fix this issue in this PR, maybe in a future PR or someone else can. I am saying however that we do not need a default button because its simply not needed,  A default button makes sense on the dashboard for widgets since you can make some drastic changes as it would take too long to manually re-arrange the dashboard. Here you just need to click the first image and then click the button.  